pkg/agent/pv_monitor_agent.go

```diff
@@ -111,9 +118,17 @@ func NewPVMonitorAgent(client kubernetes.Interface, driverName string, conn *grp
 	agent.podLister = podInformer.Lister()
 	agent.podListerSynced = podInformer.Informer().HasSynced
 
-	agent.pvChecker = handler.NewPVHealthConditionChecker(driverName, conn, client, timeout, agent.pvcLister, agent.pvLister, agent.eventRecorder)
+	//CSINode informer
+	agent.csiNodeLister = csiNodeInformer.Lister()
```

Comment:
Is this run per-node or per-cluster? If per-node, then that means every node watches every csinode object, which is expensive in large clusters, and also unnecessary since we only are interested in one csinode object?

Reply:
emmm. Yep. You are right. It is a reason we need to choose" reference environment variable" to expose node name to container